Source/WebKitLegacy/mac/Misc/WebNSDataExtras.mm

```diff
@@ -83,9 +83,9 @@ - (NSString *)_web_capitalizeRFC822HeaderFieldName
     }
     if (somethingChanged) {
         if (useUniCharPtr)
-            result = adoptCF(CFStringCreateWithCharactersNoCopy(kCFAllocatorDefault, uniCharPtr, len, nullptr)).bridgingAutorelease();
+            result = adoptCF(CFStringCreateWithCharactersNoCopy(kCFAllocatorDefault, uniCharPtr, len, kCFAllocatorDefault)).bridgingAutorelease();
```

Comment:
The description here is inaccurate. This change does not fix a bug. The backing buffer is correctly deallocated by the old code. The constant `kCFAllocatorDefault` simply expands to `nullptr`. So the code here is identical, and this is just a style change. There would be a bug here if this was using `kCFAllocatorNull`, which is not equal to `nullptr`, but it’s not.

So I understand if you want to suggest a style change. But it’s not appropriate to land that in a patch that claims to "deallocate the backing buffer".

Reply:
I had no idea. I thought kCFAllocatorNull WAS nullptr.

Reply:
Not to preach, but this is a reason testing is important, even if we think we spotted a mistake by code inspection.